File: FrontEnd/src/app/features/reservations/classes/dtos/form/passenger-read-dto.ts
import { Guid } from 'guid-typescript'
// Custom
import { NationalityDropdownVM } from 'src/app/features/nationalities/classes/view-models/nationality-dropdown-vm'
import { SimpleEntity } from 'src/app/shared/classes/simple-entity'

export interface PassengerReadDto {

    id: number
    reservationId: Guid
    gender: SimpleEntity
    nationality: NationalityDropdownVM
    occupant: SimpleEntity
    lastname: string
    firstname: string
    birthdate: string
    remarks: string
    specialCare: string
    isCheckedIn: boolean

}

File: API/ViewModels/PassengerReadDto.cs
using System;
using API.Infrastructure.Classes;

namespace API.ViewModels {

    public class PassengerReadDto {

        public int Id { get; set; }

        public Guid ReservationId { get; set; }

        public string Lastname { get; set; }
        public string Firstname { get; set; }
        public string Birthdate { get; set; }
        public string Remarks { get; set; }
        public string SpecialCare { get; set; }
        public bool IsCheckedIn { get; set; }

        public NationalityDto Nationality { get; set; }
        public SimpleEntity Gender { get; set; }

    }

    public class NationalityDto : SimpleEntity {

        public string Code { get; set; }

    }

}